Marlin/src/pins/ramps/pins_MKS_GEN_L.h

```diff
@@ -37,10 +38,15 @@
 // Heaters / Fans
 //
 
-#define MOSFET_A_PIN                          10  // HE0
-#define MOSFET_B_PIN                           7  // HE1 or FAN Hotend Cooling
-#define MOSFET_C_PIN                           8  // HBED
-#define FAN0_PIN                               9  // FAN Part Cooling
+//#define HEATER_BED_PIN                       8  // H-BED / "MOSFET_C_PIN"
```

Comment:
`// H-BED / "MOSFET_C_PIN"`

The label on the PCB is "H-BED". The `MOSFET_C_PIN` comment means that this board uses Marlin's custom "`MOSFET_C_PIN`" naming from the `pins_RAMPS.h` include. The entire line can be removed, but I left it here because this particular board's pins file keeps getting changed incorrectly.